### DISCUSSION TITLE
AI作画完成后会出现key设置错误的提示

### DISCUSSION THREAD

**Discussion by zhaolu1206** [Yue-Yang/ChatGPT-Siri#69] [General]

**Comment by Yue-Yang** [comment]
有没有截图看看

**Comment by zhaolu1206** [comment]
![PXL_20230319_011650516 MP](https://user-images.githubusercontent.com/30510452/226148613-c5ab3120-4630-4912-aef8-8244cd481103.jpg)
![PXL_20230319_011700773](https://user-images.githubusercontent.com/30510452/226148617-9fdeec53-46bf-4647-8305-cb0a25f5e824.jpg)
生成图像后，确认完成就会出现这个错误信息

**Comment by Yue-Yang** [comment]
那就不清楚了。我这边测试是iOS16，没有这个问题。

这个是没有设置key才会弹出的内容，你既然已经获取了图片。应该也不可能出这个。已经验证通过了key格式没问题才继续执行

**Comment by zhaolu1206** [comment]
> 那就不清楚了。我这边测试是iOS16，没有这个问题。
> 
> 这个是没有设置key才会弹出的内容，你既然已经获取了图片。应该也不可能出这个。已经验证通过了key格式没问题才继续执行

对啊，key肯定是没有问题的，我的ios也是16

**Comment by Yue-Yang** [comment]
你这个都不是全面屏，怎么升级iOS16的🤔️

**Comment by zhaolu1206** [comment]
> 你这个都不是全面屏，怎么升级iOS16的🤔️

确定是16啊，se2，可以升16

**Comment by zhaolu1206** [comment]
> 你这个都不是全面屏，怎么升级iOS16的🤔️

![463178F4-4CCB-4708-B302-F97803B6D0F1](https://user-images.githubusercontent.com/30510452/226173511-67341ad0-f816-49d4-aad8-2fc9c3cde217.png)

**Comment by chk970222** [comment]
同样的我也会在获取图片后返回这个错误

**Comment by chk970222** [comment]
> ![PXL_20230319_011650516 MP](https://user-images.githubusercontent.com/30510452/226148613-c5ab3120-4630-4912-aef8-8244cd481103.jpg)
> ![PXL_20230319_011700773](https://user-images.githubusercontent.com/30510452/226148617-9fdeec53-46bf-4647-8305-cb0a25f5e824.jpg)
> 生成图像后，确认完成就会出现这个错误信息

我在捷径里编辑删除错误提示这个字段就好了，你试试

**Comment by zhaolu1206** [comment]
> > ![PXL_20230319_011650516 MP](https://user-images.githubusercontent.com/30510452/226148613-c5ab3120-4630-4912-aef8-8244cd481103.jpg)
> > ![PXL_20230319_011700773](https://user-images.githubusercontent.com/30510452/226148617-9fdeec53-46bf-4647-8305-cb0a25f5e824.jpg)
> > 生成图像后，确认完成就会出现这个错误信息
> 
> 我在捷径里编辑删除错误提示这个字段就好了，你试试

OK，删除掉错误提示之后确实没有了
